### PR TITLE
Added recipe for 'wrfplot'

### PR DESCRIPTION
**Pull request by wxguy**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by wxguy** [comment]
@conda-forge/core and @conda-forge/help-python 
https://github.com/orgs/conda-forge/teams/help-python

The package successfully passed all tests two days ago but is yet to get uploaded. Request review.

**Comment by wxguy** [comment]
@conda-forge/staged-recipes 

@conda-forge-admin, please ping conda-forge/staged-recipes

Pining to get status on the successfully passed 'wrfplot' recipe. I have seen some recipes passed after this pull request was moved to the feedstock repo. Is there anything else I need to do in this pull request to make it to conda-forge?

File: recipes/wrfplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  entry_points:
+    - wrfplot=wrfplot.wrfplot:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - setuptools
+    - python >3.8
```
Comment (ocefpaf) [inline recipes/wrfplot/meta.yaml:22]: Metadata says it is:

```suggestion
    - python >=3.7
```

File: recipes/wrfplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - setuptools-scm
+    - pip
+  run:
+    - python >3.8
```
Comment (ocefpaf) [inline recipes/wrfplot/meta.yaml:26]:
```suggestion
    - python >=3.7
```

File: recipes/wrfplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - cartopy
+    - matplotlib-base <3.6
+    - wrf-python >=1.3
+    - imageio
+    - tqdm
+    - netcdf4
+    - basemap
```
Comment (ocefpaf) [inline recipes/wrfplot/meta.yaml:33]: It is not in the metadata inquirentes. Also, it is a bit odd to require both cartopy and basemap/
```suggestion
```

Comment (wxguy) [reply recipes/wrfplot/meta.yaml:33]: I am also aware that `basemap` is not required. But without `basemap` the recipe would not build and throw an error message `wrf-python` require `basemap` however no dependency is provided. It is really odd as `basemap`  is not mentioned as one of the dependencies in their feedstock recipe.

Comment (ocefpaf) [reply recipes/wrfplot/meta.yaml:33]: Did that happened in the `pip check` test? If so you can comment it out. The problem is in `wrf-python` b/c basemap is optional but pip tries to look for it anyway.

**Pull request merged**